apps/api/src/app/widgets/usecases/get-notifications-feed/get-notifications-feed.usecase.ts

```diff
@@ -8,14 +8,17 @@ export class GetNotificationsFeed {
   constructor(private messageRepository: MessageRepository) {}
 
   async execute(command: GetNotificationsFeedCommand): Promise<MessageEntity[]> {
-    return await this.messageRepository.findBySubscriberChannel(
+    const result = await this.messageRepository.findBySubscriberChannel(
       command.environmentId,
       command.subscriberId,
       ChannelTypeEnum.IN_APP,
+      command.feedId,
```

Comment:
What will be returned if the user does not have any feeds?

Reply:
All messages, the check happens in the repository :)
<img width="641" alt="Screen Shot 2022-07-06 at 12 44 39" src="https://user-images.githubusercontent.com/7778680/177522017-b60f9793-a5e2-43cb-b2f7-db7fa4b1e4a3.png">